docs/src/main/asciidoc/smallrye-graphql-client.adoc

```diff
@@ -256,6 +257,17 @@ With this REST endpoint included in your application, you can simply send a GET
 and the application will use an injected typesafe client instance to call the remote service, obtain
 the films and planets, and return the JSON representation of the resulting list.
 
+=== Logging
+
+For debbuging purpose, it is possible to log the request generated by the typesafe client and the response sent back by the server by changing the log level of the `io.smallrye.graphql.client` category to `TRACE` (See xref:logging.adoc#configure-the-log-level-category-and-format[Logging guide]).
+
+This can be achieved by adding following lines to `application.properties`:
+
+```
```

Comment:
Please use a source block here with `[source,properties]`.

We had a few issues with the Markdown style when pushing to the website.